src/GitLink/Linker.cs

```diff
@@ -112,11 +114,24 @@ public static bool Link(string pdbPath, LinkOptions options = default(LinkOption
                     }
 
                     string rawUrl = provider.RawGitUrl;
-                    if (!rawUrl.Contains("%var2%") && !rawUrl.Contains("{0}"))
+                    if (rawUrl.Contains(RevisionPlaceholder) || rawUrl.Contains(FilenamePlaceholder))
```

Comment:
Good question on the ignore case. I'm leaning against it, because the rest of the URL path is all case sensitive, so making certain segments of it case insensitive could be more confusing than helpful, IMO. But I'm not strongly against it, so if you want, I'll make the change.